addons/sale/models/sale_order_line.py

```diff
@@ -158,6 +158,10 @@ def _compute_tax_id(self):
 
     @api.model_create_multi
     def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get('display_type'):
```

Comment:
```suggestion
            if vals.get('display_type') or self._context.get('default_display_type'):
```
Maybe something like this ?

Reply:
`or self.default_get(['display_type']).get('display_type')` then (I forgot the `ir.default` in my remark, even if I believe using a default on this field is a bad idea ^^)